src/java/fr/paris/lutece/plugins/forms/web/admin/FormJspBean.java

```diff
@@ -456,6 +472,199 @@ public String getModifyForm( HttpServletRequest request ) throws AccessDeniedExc
+        int idForm = NumberUtils.toInt( request.getParameter( FormsConstants.PARAMETER_ID_FORM ), FormsConstants.DEFAULT_ID_VALUE );
+
+        if ( idConfig == FormsConstants.DEFAULT_ID_VALUE )
+        {
+            Map<String, String> mapParameters = new LinkedHashMap<>( );
+            mapParameters.put( FormsConstants.PARAMETER_ID_FORM, String.valueOf( idForm ) );
+            
+            return redirect( request, VIEW_MANAGE_EXPORT, mapParameters );
+        }
+        
+        List<FormExportConfig> existingConfigList = FormExportConfigHome.findByForm( idForm );
+        int newOrder = 0;
+        FormExportConfigHome.removeByForm( idForm );
+        for ( FormExportConfig config : existingConfigList )
+        {
+          if ( config.getId( ) != idConfig )
+          {
+              config.setOrder( ++newOrder );
+              FormExportConfigHome.create( config );
+          }
+        }
+        
+        Map<String, String> mapParameters = new LinkedHashMap<>( );
+        mapParameters.put( FormsConstants.PARAMETER_ID_FORM, String.valueOf( idForm ) );
+        
+        return redirect( request, VIEW_MANAGE_EXPORT, mapParameters );
+    }
+    
+    @Action( ACTION_MOVE_UP_EXPORT_CONFIG )
+    public String doMoveUpExportConfig( HttpServletRequest request )
+    {
+        int idConfig = NumberUtils.toInt( request.getParameter( PARAMETER_ID_CONFIG ), FormsConstants.DEFAULT_ID_VALUE );
+        int idForm = NumberUtils.toInt( request.getParameter( FormsConstants.PARAMETER_ID_FORM ), FormsConstants.DEFAULT_ID_VALUE );
+
+        if ( idConfig == FormsConstants.DEFAULT_ID_VALUE )
+        {
+            Map<String, String> mapParameters = new LinkedHashMap<>( );
+            mapParameters.put( FormsConstants.PARAMETER_ID_FORM, String.valueOf( idForm ) );
+            
+            return redirect( request, VIEW_MANAGE_EXPORT, mapParameters );
+        }
+        
+        List<FormExportConfig> existingConfigList = FormExportConfigHome.findByForm( idForm );
+        
+        FormExportConfig configMovedUp = FormExportConfigHome.findByPrimaryKey( idConfig );
+        int orderMovedUp = configMovedUp.getOrder( );
+        
+        for ( FormExportConfig config : existingConfigList )
+        {
+            if ( config.getOrder( ) == orderMovedUp - 1 )
+            {
+                config.setOrder( orderMovedUp );
+                FormExportConfigHome.update( config );
+                break;
+            }
+        }
+        configMovedUp.setOrder( orderMovedUp - 1 );
+        
+        FormExportConfigHome.update( configMovedUp );
+        
+        Map<String, String> mapParameters = new LinkedHashMap<>( );
+        mapParameters.put( FormsConstants.PARAMETER_ID_FORM, String.valueOf( idForm ) );
+        
+        return redirect( request, VIEW_MANAGE_EXPORT, mapParameters );
+    }
+    
+    @Action( ACTION_MOVE_DOWN_EXPORT_CONFIG )
+    public String doMoveDownExportConfig( HttpServletRequest request )
+    {
```

Comment:
Move up and Move down seem to b essentially the same method, except for the  `orderMovedUp` and `orderMovedDown` variables. Maybe you should factorize these two methods (perhaps you can pass a lambda to compute the `orderMovedXXX` var) ?